mizar/networkpolicy/networkpolicy_util.py

```diff
@@ -359,6 +387,11 @@ def add_affected_networkpolicy_by_pod_label(self, policy_name_list, label):
             for policy_name in networkpolicy_opr.store.label_networkpolicies_egress_store[label]:
                 policy_name_list.add(policy_name)
 
+    def add_endpoint_affected_networkpolicy_by_pod_label(self, policy_name_list, label):
```

Comment:
Here endpoint is not a noun. It's adjectives to describe networkpolicy. It's "endpoint_affected_networkpolicy". 
In a networkpolicy definition, the podSelector is a list of labels. And the podSelector can appear two places: one is in policy ingress/egress rules, the second one is in the beginning of the policy definition which means the policy's power on the pods/endpoints. Accordingly, when a pod changed label, it may affect networkpolicy ingress/egress rules, or affect networkpolicy-endpoints mapping. If a policy is affected by pod label change, I call it "endpoint affected networkpolicy".

I added above comments into the code.

Reply:
Having comments explaining the intention is helpful, if the function needs clarification. good move.